### ISSUE TITLE
Error in ClustDist-class man

### ISSUE THREAD

**Issue by lgatto** [lgatto/pRoloc#89]
```
The error most likely occurred in:

> ### Name: ClustDist-class
> ### Title: Class '"ClustDist"'
> ### Aliases: ClustDist class:ClustDist ClustDist-class
> ###   plot,ClustDist,MSnSet-method show,ClustDist-method
> ### Keywords: classes
> 
> ### ** Examples
> 
>   showClass("ClustDist")
Class "ClustDist" [package "pRoloc"]

Slots:
                                                                        
Name:           k       dist       term         id       nrow    clustsz
Class:    numeric       list  character  character    numeric       list
                            
Name:  components       fcol
Class:     vector  character
>   
>   library('pRolocdata')

This is pRolocdata version 1.11.9.
Use 'pRolocdata()' to list available data sets.
>   data(dunkley2006)
>   par <- setAnnotationParams(inputs =
+                     c("Arabidopsis thaliana genes",
+                     "TAIR locus ID"))
Using species Arabidopsis thaliana genes (TAIR10 (2010-09-TAIR10))
Using feature type TAIR locus ID(s)
Connecting to Biomart...
>                     
>   ## add protein set/annotation information                  
>   xx <- addGoAnnotations(dunkley2006, par)
Loading required namespace: GO.db

Loading required package: GO.db
Error in `colnames<-`(`*tmp*`, value = character(0)) : 
  attempt to set 'colnames' on an object with less than two dimensions
Calls: addGoAnnotations -> colnames<- -> colnames<-
Execution halted
```
Haven't looked at it at all so far.

**Comment by lgatto** [comment]
Fails because no GO terms are fetched in `makeGoSet` and `getGOFromFeatures`.

**Comment by lgatto** [comment]
Fixed in da9eb61
